src/style.css

```diff
@@ -1 +1,66 @@
-/* Styles go here */
+/* stylelint-disable rule-empty-line-before */
+* {
+  box-sizing: border-box;
+}
+html {
+
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  --color-background-header: #fff;
+  --color-link-active: #00acdc;
+  --color-link: #000;
+}
+body {
+  margin: 0 auto;
+}
+.header {
```

Comment:
Add 'width: 100%;' and 'max-width: 1024px;' to the .header to make sure the layout fits on 1024px.